### PR TITLE
Add SHA and link to "what will be deployed"-diff

### PR DESCRIPTION
**Pull request by Wind0r**
I hope its done as wished in #11

### CODE REVIEW

**Comment by fabrik42** [comment]
Very nice, thank you! Add yourself to the [changelog](https://github.com/applikatoni/applikatoni/blob/master/CHANGELOG.md) file and I will merge this.


Cheers,
Christian

File: server/assets/applikatoni.js [hunk 1]
```diff
@@ -37,6 +37,10 @@ Commit.prototype.updatedAt = function() {
   return this.rawJson.commit.committer.date;
 }
 
+Commit.prototype.html_url = function() {
```
Comment (mrnugget) [inline server/assets/applikatoni.js]: Could you use `camelCase` here? That's the standard we try to follow with JS :)

File: server/github.go [hunk 2]
```diff
@@ -16,9 +16,10 @@ import (
 const gitHubAPI = "https://api.github.com"
 
 type GitHubCommit struct {
-	Author *models.User `json:"author"`
-	Sha    string       `json:"sha"`
-	Commit struct {
+	Author   *models.User `json:"author"`
+	Sha      string       `json:"sha"`
+	HTML_URL string       `json:"html_url"`
```
Comment (mrnugget) [inline server/github.go]: How about `HtmlURL` here? That would match the [naming here](https://github.com/Wind0r/applikatoni/blob/f1f5cef287d5989ae2e1b6a2e6df68466f853ffd/server/github.go#L48-L53).

**Comment by mrnugget** [comment]
Alright, I'll merge this now! Thanks a lot @Wind0r :)

**Pull request merged**